### PR TITLE
:sparkles: tutorial mode

### LINKED ISSUE #411: user guide for the first connexion
As a user of skillz
I want to have an introduction on the tool
So that I can understand the tool and use it easily

Rules

Scenario 1
Given it is my first connexion on Skillz
When I arrived in the landing page
Then I have a user guide which have to present me how to use Skillz

Scenario 2
Given it is not my first connexion on Skillz
When I arrived on the landing page
Then I can see a button "guide"
And when I click on it, it guide my to use properly Skillz

### PR DESCRIPTION
**Pull request by MailyLehoux**
resolve #411

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,15 +39,19 @@
...
     "object-path": "^0.11.7",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.7.1",
+    "react-joyride": "^2.5.3",
     "react-responsive": "9.0.2",
     "react-swipeable": "6.1.2",
     "recharts": "^2.3.2",
+    "sharp": "^0.30.7",
+    "styled-components": "^4.4.1",
```
Comment (benjaminbrgs) [inline package.json:54]: Why do we need mkdirp-classic, sharp and styled-component ?

File: src/components/organisms/AddSkilListSelector.tsx [hunk 2]
```diff
@@ -65,30 +66,56 @@ const AddSkillListSelector = ({
...
+  const tourConfig = [
+    {
+      target: ".step1-infos",
+      content: t("onboarding.demo.mine.step3"),
+      title: t("onboarding.demo.mine.titlestep3"),
+    },
+    {
+      target: ".step2-add-skill",
+      content: t("onboarding.demo.mine.step4"),
+      title: t("onboarding.demo.mine.titlestep4"),
```
Comment (benjaminbrgs) [inline src/components/organisms/AddSkilListSelector.tsx:78]: I suggest to have the same step number between target and content/title

File: src/components/organisms/HomePanel/HomePanel.tsx [hunk 3]
```diff
@@ -71,174 +72,218 @@ const HomePanel = ({
...
+    },
+  ];
+
+  const tourConfigNoSkill = [
+    {
+      target: ".step3-graph",
+      content: t("onboarding.demo.zenika.step3"),
+      title: t("onboarding.demo.zenika.titlestep3"),
+    },
+  ];
```
Comment (benjaminbrgs) [inline src/components/organisms/HomePanel/HomePanel.tsx:101]: Maybe import all tour step from another file. This avoid overloading a file component

File: src/pages/index.tsx [hunk 4]
```diff
@@ -82,6 +85,7 @@ const Home = () => {
   }
   return (
     <CommonPage page={"Home"} backBar={false}>
+      <TopBar />
```
Comment (benjaminbrgs) [inline src/pages/index.tsx:88]: Rename component as TutorialTopBar otherwise it could be confused with another topbar

**Review by benjaminbrgs — CHANGES_REQUESTED** [review]
That looks very good, gg 👏🏼

**Comment by benjaminbrgs** [comment]
@MailyLehoux Button are always in English
image

File: package.json (hunk 1 above)
Comment (MailyLehoux) [reply package.json:54]: thought it as dependency of react-tour but I removed.it's useless now i'll delete it

**Review by benjaminbrgs — APPROVED** [review]
GG 💯

**Pull request merged**